woocommerce/woocommerce-admin
# scroll import action to import section of settings page

Fixes #2243

This PR adds an import query arg to the Import Historical Data note. When the query arg is present the settings screen scrolls to the Import section.

It introduces a `<ScrollTo> ... </ScrollTo>` component with an offset attribute that can be optionally used as an adjustment to the top of the component. The top of the component is the default scroll to position.

### Detailed test instructions:

- Re-create the `get-started` note using the code in this PR.
- Open the Activity panel & Notes.
- Click on the import historical data `Get Started`
- The setting screen should scroll to the import section

### Changelog Note:

Tweak: have Import note action scroll to import section of settings screen.
